src/oaklib/implementations/pronto/pronto_implementation.py

```diff
@@ -520,6 +520,9 @@ def _clean(v: str) -> str:
                 if syn.description == patch.old_value:
                     syn.description = patch.new_value
         elif isinstance(patch, kgcl.NewSynonym):
+            t = self._entity(patch.about_node, strict=True)
+            # Get scope from patch.qualifier 
+            # rather than forcing all synonyms to be related.
             scope = str(patch.qualifier.value).upper() if patch.qualifier else "RELATED"
             t.add_synonym(description=patch.new_value, scope=scope)
 
```

Comment:
The line 523 was already present before. For some reason the reverting of https://github.com/INCATools/ontology-access-kit/pull/215 messed things up. The only reason I have this PR is to bring the change in lines 526 and 527 to Chris' attention.